handlers/config.go

```diff
@@ -210,61 +200,57 @@ type KialiCrippledFeatures struct {
 	ResponseTimePercentiles bool `json:"responseTimePercentiles"`
 }
 
-func CrippledFeatures(w http.ResponseWriter, r *http.Request) {
-	defer handlePanic(w)
-
-	requiredMetrics := []string{
-		"istio_request_bytes_bucket",
-		"istio_request_bytes_count",
-		"istio_request_bytes_sum",
-		"istio_request_duration_milliseconds_bucket",
-		"istio_request_duration_milliseconds_count",
-		"istio_request_duration_milliseconds_sum",
-		"istio_requests_total",
-		"istio_response_bytes_bucket",
-		"istio_response_bytes_count",
-		"istio_response_bytes_sum",
-	}
+func CrippledFeatures(client prometheus.ClientInterface) http.HandlerFunc {
```

Comment:
I guess "Crippled" is now a discouraged word.  Maybe we should change occurrences to `Disabled` or `Unavailable`.